app/views/repairs/_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_for @repair do |f| %>
+    <div>
+   Repair Type: <%= f.text_field :repair_type, class:'form-control', placeholder:'What type of repair will you like done' %>
+    </div>
+    <div>
```

Comment:
Your form should not contain multiple `div`s. You can add `<br/>` to move the form contents onto separate lines, like in the [example here](https://api.rubyonrails.org/v5.1/classes/ActionView/Helpers/FormHelper.html).